README.md

```diff
@@ -20,6 +20,14 @@ export HOMEBREW_CORE_GIT_REMOTE="..."  # put your Git mirror of Homebrew/homebre
 
 The default Git remote will be used if the corresponding environment variable is unset.
 
+If you want to run the Homebrew installer non-iteratively without prompting for passwords (e.g. in automation scripts), you can use:
+
+```bash
+NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+The non-interactive behaviour will be enabled when the environment variable `NONINTERACTIVE` is set to a non-empty value.
+
```

Comment:
```suggestion

```

Think this is implied by the above.

Do you think this should be on the https://brew.sh page cough?

Reply:
> Do you think this should be on the https://brew.sh page cough?

I think only advanced users have the use-case to run the installer non-interactively. They can find this GitHub repository by the script URL <https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh>. Advanced users will look into the installer content before adding it to their own scripts, right?

Reply:
@XuehaiPan Yup, makes sense 👍🏻